### ISSUE TITLE
[Feature Request] Аналитика: состав портфеля

### ISSUE THREAD

**Issue by 3XclusiVe** [softandiron/tinkproject#62]
Предлагаю рассмотреть возможность добавления дополнительных аналитических данных по составу портфеля.
По отраслям, компаниям, а также странам с учетом компаний в фондах.

**Comment by alexanderkmd** [comment]
Такое сделать возможно, но будет реализовано только в ветке [API2](https://github.com/softandiron/tinkproject/tree/API2)
Новый API тинькофф выдает данные о секторе и стране актива.

Только вот с составом в фондах - у вас есть какой-либо публичный API выдающий такие данные?

**Comment by 3XclusiVe** [comment]
Да, действиетльно, похоже с составом фондов api тинькофф не может работать, хотя в UI у них это есть. 

Из публичного апи нашел только Yahoo 
https://github.com/dpguthrie/yahooquery (там есть методы fund_sector_weightings и fund_holding_info)
Но работает далеко не для всех фондов, например SPY распознает, а вот с фондами finex уже проблемы.
